www_src/pages/page/page.jsx

```diff
@@ -92,7 +94,12 @@ var Page = React.createClass({
         </div>
       </div>
 
-      <div className={classNames({overlay: true, active: this.state.showAddMenu})} onClick={this.toggleAddMenu}/>
+      // Overlay to show when elements are placing themselves on the page
```

Comment:
React will treat this as literal text. Better to just not put it in, since the markup is fairly clear in what it's for.